### PR TITLE
Replace Antox client with aTox

### PR DESCRIPTION
**Pull request by nurupo**
Antox hasn't been maintained for a long time now.

aTox is a new Android client developer by one of the core Tox developers.

### CODE REVIEW

**Review by robinlinden — APPROVED** [review]
LGTM!

File: themes/website/static/js/osdetect.js [hunk 1]
```diff
@@ -82,23 +82,11 @@ if (window.navigator.userAgent.indexOf("Android") != -1) {
...
-		icon: "external-link",
-		desc: true,
-		dlLink: "https://play.google.com/store/apps/details?id=chat.tox.antox",
-	}, {
-		title: "Antox APK",
-		name: "antox",
-		icon: "download",
-		desc: true,
-		dlLink: "https://pkg.tox.chat/fdroid/repo/antox.apk",
+		dlLink: "https://github.com/evilcorpltd/aTox/releases/download/v0.5.1/atox.apk",
```
Comment (GrayHatter) [inline themes/website/static/js/osdetect.js:89]: do you really mean to version lock it to v0.5.1? Seems like it'll go out of date both quickly and often

**Review by GrayHatter — CHANGES_REQUESTED** [review]

File: themes/website/static/js/osdetect.js (hunk 1 above)
Comment (nurupo) [reply themes/website/static/js/osdetect.js:89]: Well, you can't use `latest` for direct file downloads, e.g. `https://github.com/evilcorpltd/aTox/releases/download/latest/atox.apk` is a HTTP 404.

What do you suggest to do instead?

Comment (nurupo) [reply themes/website/static/js/osdetect.js:89]: Alright, let's merge this as it is and change the URL later when you reply with what you had on your mind. Doesn't make much sense to stall the PR for this.

**Pull request merged**